index.html

```diff
@@ -2250,6 +2329,36 @@ <h1>Core Representations</h1>
 for more information.
     </p>
 
+    <section>
+      <h2>Creating Representations</h2>
+      <p>
+The <a href="#data-model">data model</a> provided in this specification
+supports being serialized into a variety of existing representations. Some
+applications might require the creation of a new representation. All
+representations require the following:
+      </p>
+      <ol>
+        <li>
+A representation MUST define an unambiguous encoding and decoding for all
+property names and their associated values as defined in this specification.
+This enables anything that can be represented in the <a>DID document</a> data
+model to also be represented in a compliant representation.
+        </li>
+        <li>
+The representation MUST be associated with an IANA-registered MIME type.
```

Comment:
```suggestion
The representation MUST be uniquely associated with an IANA-registered MIME type.
```

Reply:
just to clarify, would this mean that if a representation is already associated with `application/did+foo` then no other representation could come along and use that mime type?

Reply:
Yes, @jricher's change is correct and yes, that means that each MIME type is associated with one set of representation rules.